image-viewer.py

```diff
@@ -1,35 +1,50 @@
+from operator import itemgetter
 from renderer import Renderer
 from menu import AppMenu
 from tkinter import Tk, Label, Button
 from PIL import Image, ImageTk
+import json
+
 
 class ImageViewer(Renderer, AppMenu):
     def __init__(self):
-        
+        # Import configuration.json file, reads it and store data in variables
```

Comment:
I'd say this comment isn't needed since the code you've written explains itself :)
```suggestion
        
```

Reply:
Done